src/browser/page-switch.js

```diff
@@ -31,17 +29,26 @@ define((require, exports, moudle) => {
         }
       }
     }, [
-      DOM.span({
+      DOM.div({
         key: 'thumbnail',
         className: 'tab-thumbnail',
-        style: {backgroundImage: readThumbnailURI(webViewerCursor.get('location'))},
-      }),
+      }, [
+        DOM.img({
+          key: 'image',
+          className: 'image',
+          src: thumbnail,
+          alt: '',
+          onLoad: event => URL.revokeObjectURL(event.target.src),
```

Comment:
In fact if you google about blob urls you'll find stuff like this
http://blogs.adobe.com/webplatform/2012/01/17/displaying-xhr-downloaded-images-using-the-file-api/